bin/ch-umount

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+libexec="$(cd "$(dirname "$0")" && pwd)"
+. "${libexec}/base.sh"
+
+# shellcheck disable=SC2034
+usage=$(cat <<EOF
+Unmount Charliecloud SquashFS file in MOUNTDIR, removes MOUNTDIR on success
+
+Usage:
+
+  $ $(basename "$0") MOUNTDIR
+
+EOF
+)
+
+parse_basic_args "$@"
+
+if [ "$#" -ne 1 ]; then
+    usage
+fi
+
+mountdir=$1
+if ( fusermount -u "$mountdir" ); then
+    rm -rf --one-file-system "$mountdir"
+    echo "unmounted and removed ${mountdir}"
+fi
```

Comment:
Nothing will be unmounted, and the remove step will not execute. The only reason I know of that the unmount would fail is you pointed it to the wrong path. This way the directory doesn't get removed if fusermount fails.

Reply:
Does `fusermount` print a good error message?

Reply:
`$ fusermount -u test`
`fusermount: entry for /home/rgoff/devel/charliecloud/test not found in /etc/mtab`

Reply:
Do we want to create additional output or replace the output of fusermount?